src/dialect/mssql.rs

```diff
@@ -116,7 +116,17 @@ impl Dialect for MsSqlDialect {
         true
     }
 
-    fn is_column_alias(&self, kw: &Keyword, _parser: &mut Parser) -> bool {
+    fn is_column_alias(&self, kw: &Keyword, parser: &mut Parser) -> bool {
+        // if we find maybe whitespace then a newline looking backward, then `GO` ISN'T a column alias
+        // if we can't find a newline then we assume that `GO` IS a column alias
+        if kw == &Keyword::GO
+            && parser
+                .expect_previously_only_whitespace_until_newline()
+                .is_ok()
```

Comment:
if the intent is for the condition to return a bool, then we can update this function to return a bool instead of a result (where the latter is flagging an error to be handled)

Reply:
In this case, it's most useful to know if there was previously only whitespace until a newline. However in the other call of this function in parse_go, we legitimately do want a parser error. Therefore we have choices:

1. expect_previously_only_whitespace_until_newline raises a parse error which makes sense in parse_go but is converted to a boolean here
2. expect_previously_only_whitespace_until_newline returns a boolean which makes sense here and then parse_go needs to separately raise a parse error

Which is the best choice for the time being?

Reply:
Ah I see, 2. sounds desirable to return a boolean, in that the function is only checking if a condition holds, then interpretation of the condition is left to the caller of the function. in the other usage the caller does e.g.
```rust
if !self.prev_whitespace_only_is_newline() {
    return self.expected("newline ...")
}
```

Reply:
Gotcha, I will update it to do that

Reply:
Done 👍